### PR TITLE
Add section about black to contributing

### PR DESCRIPTION
**Pull request by FabioRosado**
# Description

I've noticed that a few contributors have been having issues with black. I got to admit that event I got confused the first time I tried to commit a change.

This PR adds a section about black to contributing and what you should look for when committing your changes.

I tried to not write too much about it, let me know if this makes sense or if you would like me to change something 😄 👍

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -72,6 +72,48 @@ Running the tests
 docker run --rm -ti -v $(pwd):/usr/src/app opsdroid/opsdroid:myfeature tox
 ```
 
+## Automatic Linting with Black
+
+Opsdroid is running black to deal with linting issues. Black will be triggered when you `git commit` your changes.
```
Comment (jacobtomlinson) [inline docs/contributing.md]: You actually need to set up [precommit](https://pre-commit.com/) in your development environment for this to happen. Might be worth documenting that.

The linting CI also now includes black checks.

**Comment by FabioRosado** [comment]
Thats a good point, I need to check how I activated the black prehooks since I don't remember what I did haha

**Comment by FabioRosado** [comment]
I've updated the documentation, I'm not sure if it makes sense since I don't remember how i set up everything haha

**Comment by jacobtomlinson** [comment]
Awesome thanks!

**Pull request merged**